engine/apps/api_for_grafana_incident/serializers.py

```diff
@@ -2,17 +2,31 @@
 
 from rest_framework import serializers
 
-from apps.alerts.models import AlertGroup
+from apps.alerts.models import Alert, AlertGroup
 from common.api_helpers.mixins import EagerLoadingMixin
 
 logger = logging.getLogger(__name__)
 
 
+class AlertSerializer(EagerLoadingMixin, serializers.ModelSerializer):
```

Comment:
`EagerLoadingMixin` is not used here and can be removed.

It is not needed in AlertGroupSerializer below as well, as it doesn't work without PREFETCH_RELATED or SELECT_RELATED properties.